partners/scientific-communities.md

```diff
@@ -1,38 +1,47 @@
-step that adds your specific requirements for package affiliation.
 
-Through this single end-to-end review process, a package can potentially:
+:::{figure-md} pyopensci-3-pillars
 
-- Be accepted by pyOpenSci.
-- Undergo a fast-track review and get published with JOSS (if in scope).
-- Achieve affiliation with your domain-specific community.
+<img src="../images/pyopensci-pillars-flower.png" alt="Flower" width="500">
 
-Here we discuss our process for
-supporting this community affiliation partnership.
+pyOpenSci has three core programs. Community partnerships is a program that allows domain specific scientific Python communities to leverage our peer review process rather than creating their own process. Through this partnership maintainers can become affiliated with both a domain specific community, published in the Journal of Open Source Software and also become a part if the pyOpenSci community.
+:::
 
-:::{figure-md} pyos-badge-home
-:class: myclass
+Numerous domain-specific scientific Python communities are developing and maintaining their own peer review
+processes to track and recommend packages relevant to their specific area(s) of expertise.
+However, running a high quality peer review process takes time and resources.
 
-<img src="../images/peer-review-partners-process.png" alt="Diagram showing the pyOpenSci peer review process. At the top there are 4 boxes labeled submit package for review, package review, package accepted and fast track in JOSS. Each of these are steps in our review process. At the bottom of the diagram are yellow boxes highlighting the fact that in the package review, an additional layer of review is added that contains criteria for a package to become affiliated with a specific community. The outcome is that a package can become a part of the pyOpenSci ecosystem, published in JOSS and also community affiliated with a single end-to-end review process.">
+pyOpenSci offers a program where communities can partner with us and leverage our peer review process, which is
+built on community-driven packaging guidelines and standards. Through this partnership, we enhance our standard review process with a community-specific
+that integrates your specific requirements for package affiliation.
 
-Scientific communities can leverage our peer review process by adding a layer of review associated with requirements each community has for packages to be labeled as "affiliated" with that specific community. To see an example of such requirements, look at the [Pangeo partner page in this guide](pangeo).
+:::{figure-md} pyopensci-partners
+
+<img src="../images/peer-review-partners-process.png" alt="." width="500">
+
+Through this partnership, we add an additional layer of checks to the pyOpenSci peer review process that align with a domain-specific community's needs. Often these checks are around packages using existing core functionality of community tools and adhering to community-specific data structure formats.
```

Comment:
Ahhh @kierisi i see where that word customized came from. i used it in this graphic. i'll revisit that word as it's bothering me now. maybe customized review step? rather than the entire review being custom?